prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -6,6 +6,8 @@ command -v zsh > /dev/null 2>&1 && MYDIR=$(dirname $(zsh -c 'echo ${0:A}' "$0"))
 test -z ${MYDIR+x} && MYDIR="$(dirname $(readlink -f $0))"
 source $MYDIR/setenv.sh
 
+if [ -z $FILEWORKDIRRUN ]; then FILEWORKDIRRUN=$FILEWORKDIR; fi              # directory where to find the run-related files (grp, collision context)
```

Comment:
Why do we need 2 directories, what is the difference between these two?

Reply:
This is because the GRP and collision context are run dependent, while the geometry, dictionaries... are not. E.g. on the gird, for the processing of the pilot beam data, we can copy only 1x the files that are in common.

Reply:
Aren't we going to upload external files per job as we were doing in run2? OK, if we read from predefined directories, this change makes sense.

Reply:
Sorry, I said it wrong, what I meant is when we are not on the grid, one does not need to copy the common files multiple times. On the grid they will be copied on the worker node anyway - I would prefer to avoid reading remotely.

Reply:
I think for now it is ok, eventually we should anyway read most objects from the CCDB, then we don't need this any more.
Instead of creating copies locally, one could also just link the files locally and then work with a single `FILEWORKDIR`. That might avoid confusion. E.g. the matlut.root file is optional. If one puts it to the wrong folder, it will not be used.

Reply:
Yes, one could also just link the run-dependent files locally, and keep the rest in the common folder - this is how I do it actually, e.g. (file names are fake):
```
commonFolder/grp_505548.root
commonFolder/grp_505673.root
commonFolder/geometry.root
cd myFolder
ln -s commonFolder/grp_505673.root grp.root
```
then I set:
``` 
FILEWORKDIR=commonFolder
FILEWORKDIRRUN=myFolder
```

Reply:
Left as it is.